Code/Tools/FBuild/FBuildTest/Tests/TestIf.cpp

```diff
@@ -23,6 +23,8 @@ class TestIf : public FBuildTest
     void IfFunctionBool() const;
     void IfFunctionInt() const;
     void IfFunctionStringCompare() const;
+    void IfFunctionSet() const;
```

Comment:
IfFunctionSet is defined and implemented, but there is no REGISTER_TEST so the test is not active.

Reply:
Fixed in 98ef95275848f20cea086e8c02cc3e13c8657003.